source/data/local_migration.rst

```diff
@@ -0,0 +1,236 @@
+.. _migrating_local:
+
+************************************
+Migrating Data Between Local Systems
+************************************
+
+.. note::
+
+    Large scale data migration can be challenging and time consuming. Please
```

Comment:
Somewhere at the top, it will be good to say "Avoid using the mv command to move directories that will result in moving a large number of files/directories".
Also, this particular one refers to Jet, but the general guidelines apply for Hera/Niagara/Jet.